### PR TITLE
Client tasks

### PR DESCRIPTION
**Pull request by hammerfunk**
Updated copy and fixed styling
Support letters
PDFs
Success Emails

### CODE REVIEW

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ def promotion_step4
 
           hint "How does the letters of support system work?", %(
             <p>
-              First, you should enter details of your chosen supporters below. We will then email them a link to a web form, where they can enter their letter of support and click on the ‘Send support request’ button
+              First, you should enter details of your chosen supporters below. We will then email them a link to a web form, where they can enter their letter of support and click on the ‘Send support request’ button.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb]: Line is too long. [217/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb [hunk 2]
```diff
@@ -131,7 +131,7 @@ def promotion_step4
               Letters of support should be around 500 words, making it clear why a nomination is being supported and NOT simply saying “I wish to support the nomination of ….”.
             </p>
             <p>
-              The Queen’s Award for Enterprise Promotion recognises those who have gone above and beyond their duties promoting enterprise and entrepreneurial skills, so your letter should focus mainly on the nominee’s achievements outside of the requirements of their paid job, using your first hand knowledge of these where possible.
+              The Queen’s Award for Enterprise Promotion recognises individuals who go beyond the limits of their day-to-day role to foster an entrepreneurial spirit and promote enterprise within others, so your letter should focus mainly on the nominee’s achievements outside of the requirements of their paid job, using your first hand knowledge of these where possible.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb]: Line is too long. [372/100]

File: app/pdf_generators/qae_pdf_forms/general/draw_elements.rb [hunk 3]
```diff
@@ -110,21 +113,19 @@ def render_urn
...
       award_title = "Queen’s Award for Enterprise Promotion #{form_answer.award_year.year}"
     else
-      award_title = form_answer.decorate.award_application_title
+      award_title = form_answer.decorate.award_application_title_print
     end
-    text_box award_title.upcase,
-             header_text_properties.merge(at: [32.mm, 130.mm + DEFAULT_OFFSET],
-                                          style: :bold)
+    text award_title.upcase,
+             header_text_properties.merge(style: :bold)
```
Comment (matthewford) [inline app/pdf_generators/qae_pdf_forms/general/draw_elements.rb]: Align the parameters of a method call if they span more than one line.

File: app/pdf_generators/qae_pdf_forms/general/draw_elements.rb [hunk 4]
```diff
@@ -110,21 +113,19 @@ def render_urn
...
+  end
+
+  def render_company_name
+    text "<b>#{form_answer.decorate.company_name.try(:upcase)}</b>",
+        header_text_properties.merge(inline_format: true)
```
Comment (matthewford) [inline app/pdf_generators/qae_pdf_forms/general/draw_elements.rb]: Align the parameters of a method call if they span more than one line.

**Comment by RuslanHamidullin** [comment]
Good to merge :+1:  gonna check rspecs

**Pull request merged**